prow/plank/controller.go

```diff
@@ -318,7 +319,9 @@ func (c *Controller) syncJenkinsJob(pj kube.ProwJob, reports chan<- kube.ProwJob
 			pj.Status.State = kube.SuccessState
 			pj.Status.Description = "Jenkins job succeeded."
 			for _, nj := range pj.Spec.RunAfterSuccess {
-				if _, err := c.kc.CreateProwJob(NewProwJob(nj)); err != nil {
+				npj := NewProwJob(nj)
+				npj.Status.ParentName = pj.Metadata.Name
```

Comment:
We want to know the output from a particular run though? We want to pass on the results of a successful job to the children that are started after success so that they can use that version of the results (?)